packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -172,78 +226,68 @@ function NavTab({ icon: TabIcon, name, path }) {
 }
 
 function MobileNavTabs() {
+  const { isNarrowWidth } = useResponsive();
   return (
     <div
       style={{
         backgroundColor: 'white',
         borderTop: `1px solid ${colors.n10}`,
         bottom: 0,
         ...styles.shadow,
-        display: 'flex',
+        display: isNarrowWidth ? 'flex' : 'none',
         height: '80px',
         justifyContent: 'space-around',
         paddingTop: 10,
         width: '100%',
       }}
     >
-      <NavTab name="Budget" path="/budget" icon={Wallet} isActive={false} />
-      <NavTab
-        name="Accounts"
-        path="/accounts"
-        icon={PiggyBank}
-        isActive={false}
-      />
-      <NavTab name="Settings" path="/settings" icon={Cog} isActive={false} />
+      <NavTab name="Budget" path="/budget" icon={Wallet} />
+      <NavTab name="Accounts" path="/accounts" icon={PiggyBank} />
+      <NavTab name="Settings" path="/settings" icon={Cog} />
     </div>
   );
 }
 
-class FinancesApp extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { isMobile: isMobile() };
-    this.history = createBrowserHistory();
+function FinancesApp(props) {
+  const patchedHistory = createBrowserHistory();
```

Comment:
This would fix my concerns about when the patching is applied, and also avoid recreating the history object on each render:
```suggestion
  const [patchedHistory] = useState(() => createBrowserHistory());
```